### ISSUE TITLE
[T3] Elipse

### ISSUE THREAD

**Issue by Coolgatty** [IIC3253/2023#32]
Podemos asumir que x1 != x2 e y1 != y2?

**Comment by raimundomartinez** [comment]
No, de hecho al hacer "exponenciación" de g, la primera adición corresponde a g * g en donde se cumple que x1==x2 y y1==y2.
